src/test/regress/pg_regress_multi.pl

```diff
@@ -261,6 +265,12 @@ sub revert_replace_postgres
 # partial run, even if we're now not using valgrind.
 revert_replace_postgres();
 
+my $host = "localhost";
+my $user = "postgres";
+my $dbname = "postgres";
+my $realWorker1Host = "localhost";
+my $realWorker2Host = "localhost";
```

Comment:
What does "real" mean here? Does it mean "public address" (which is accessible from the outside world) or "private address" (local to the cluster)

Maybe change this name and other places "real" have been used so it is easier to understand when to use this.